components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -1110,22 +1102,13 @@ private String getNextStringInLexicalOrder(String str) {
         return new String(bytes);
     }
 
-    private OMElement queryDatabase(String columnFamily,
-                                    QueryServiceStub.CompositeIndex[] compositeIndex)
-            throws APIMgtUsageQueryServiceClientException {
+    private OMElement queryDatabase(String columnFamily) throws APIMgtUsageQueryServiceClientException {
```

Comment:
I didn't add this method it was already there when i did the changes. i just removed one of the parameters since it was no longer needed.